src/sentry/partnerships/base.py

```diff
@@ -0,0 +1,19 @@
+from collections.abc import Sequence
+
+from sentry.relay.types import RuleCondition
+from sentry.utils.services import Service
+
+
+class Partnership(Service):
+    """
+    Partnership handles the partnership between the Sentry org and a partner.
+    Based on the partnership agreementsome behavior is enabled or disabled.
+    """
+
+    __all__ = "get_inbound_filters"
```

Comment:
```suggestion
    __all__ = ("get_inbound_filters",)
```
i think this should fix the failing tests